app/model/exchange/passmarksettings/PassMarkSettings.scala

```diff
@@ -38,6 +39,34 @@ object Phase1PassMarkSettings {
   import repositories.BSONDateTimeHandler
   implicit val phase1PassMarkSettingsFormat = Json.format[Phase1PassMarkSettings]
   implicit val phase1PassMarkSettingsHandler = Macros.handler[Phase1PassMarkSettings]
+
+  def merge(latestPassMarkSettings: Option[Phase1PassMarkSettings],
+            newestPassMarkSettings: Phase1PassMarkSettings): Phase1PassMarkSettings = {
```

Comment:
I'd call this "newPassMarkSettings" just so no one gets confused between "latest" and "newest"